DSP_Plugins.GalacticScale/Scripts/PatchPlanetSize/PatchOnPlanetRawData.cs

```diff
@@ -162,5 +162,39 @@ public static bool InitModData(byte[] refModData, ref PlanetRawData __instance,
 
             return false;
         }
+        //[HarmonyPrefix]
```

Comment:
Why is this one commented ?


Plus use : /* */ to comment instead of // every lines

Reply:
Because its not doing anything :)
and the // is because thats how the selection comment function I added to vs works :)